__tests__/app.test.js

```diff
@@ -261,25 +261,91 @@ describe("app core POST requests", () => {
 
     describe("POST /api/articles/:article_id/comments", () => {
     //Functionality tests
+        test("Status: 201 should return the posted comment", () => {
+            //test variable for parametric endpoint
+            const article_id = 3;
+            //test comment for post request
+            const newComment = {
+                "username": "lurker",
+                "body": "I don't know where I am."
+            };
+            return supertest(app)
+            .post(`/api/articles/${article_id}/comments`)
+            .send(newComment)
+            .then((result)=> {
+                expect(result.status).toBe(201);
+                const comment = result.body.comment;
+                expect(comment.author).toBe(newComment.username);
+                expect(comment.body).toBe(newComment.body)
+            })
+        })
```

Comment:
Good test Karl. I'd recommend removing the comments once they've served their purpose for you - I can see from the code what it's doing. Great to see you asserting the response body values are exactly the ones you submitted.